### DISCUSSION TITLE
Confused about the non-exist image tag in examples

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [apache/skywalking#11438] [General]
Like the following example YAML file. It gives the `latest` docker image tag which is not exist. This is not a big problem. But people still need to change it everytime. Considering I've fixed a few minor issues like this, it might not be a good practice to do this over and over.

https://github.com/apache/skywalking-rover/blob/a2c369331c0b0f56b464429ca2e6d18d310ec571/docs/en/setup/examples/cases/kubernetes-process/rover-daemonset.yml#L64

## Open discussion
* In my own view, providing a always working examples in guide will be friendly to those newcomers
  * Of course, the community actually does not need to meet this need. It's good to have.
* Solution 1, provide a `latest` tag for these docker images
* Solution 2, change the image tag once a new version released. This is a hard way, and difficult to make sure that.

I'm looking forward to hear your thought. Please comment below or a link if this is a duplicated topic. Thanks for your time to read this.

**Comment by wu-sheng** [comment]
Ping @mrproliu

**Comment by LinuxSuRen** [comment]
Addition: the [document](https://skywalking.apache.org/docs/skywalking-rover/next/en/setup/examples/cases/kubernetes-process/readme/) is pretty clear about what should be updated.

**Comment by mrproliu** [comment]
Thanks @LinuxSuRen, I think we can pushing the latest tag into the docker hub. Would you like to create a PR to fix it?

**Reply by wu-sheng** [reply]
Is this a docker hub release? If so, you just need to tag one fore last release as well.

Do you ask for release script change? @mrproliu

**Reply by LinuxSuRen** [reply]
Happy to do that.

**Reply by mrproliu** [reply]
Yes, I think we only need to modify the release script to include a new tag during the release process.

**Reply by LinuxSuRen** [reply]
I don't familiar with the ASF release workflow. I'm wondering if it's correct that I need to modify the following file.

https://github.com/apache/skywalking-rover/blob/a2c369331c0b0f56b464429ca2e6d18d310ec571/scripts/build/docker.mk#L35

**Reply by mrproliu** [reply]
You're right. You should add the `latest` tag in this file.
